src/main/java/org/ibp/api/rest/study/StudyInstanceResource.java

```diff
@@ -0,0 +1,52 @@
+package org.ibp.api.rest.study;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.ibp.api.domain.study.StudyInstance;
+import org.ibp.api.java.study.StudyInstanceService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import javax.annotation.Resource;
+import java.util.List;
+
+@Api(value = "Study Instance Services")
+@Controller
+@RequestMapping("/crops")
+public class StudyInstanceResource {
+
+	@Resource
+	private StudyInstanceService studyInstanceService;
+
+	@ApiOperation(value = "Create new study instance",
+		notes = "Create new study instance")
+	@RequestMapping(value = "/{cropname}/studies/{studyId}/instances/{instanceNumber}", method = RequestMethod.PUT)
```

Comment:
We don't need to expose instanceNumber as parameter since it should always just append to the last existing instance
We can make this as POST request and the URL like /{cropname}/studies/{studyId}/instances/generation

Reply:
Done